examples/bergamo_session.py

```diff
@@ -0,0 +1,77 @@
+"""Example of how to use the bergamo session module"""
+from datetime import datetime
+from pathlib import Path
+
+from aind_metadata_mapper.bergamo.session import (
+    BergamoEtl,
+    RawImageInfo,
+    UserSettings,
+)
+
+# Check the UserSettings class for list of defaults that may need to override
+# This example just sets the required fields.
+
+user_settings = UserSettings(
+    experimenter_full_name=["John Smith"],
+    subject_id="12345",
+    session_start_time=datetime(2020, 10, 10, 12, 5, 00),
+    session_end_time=datetime(2020, 10, 10, 13, 5, 00),
+    stream_start_time=datetime(2020, 10, 10, 12, 6, 00),
+    stream_end_time=datetime(2020, 10, 10, 13, 4, 00),
+    stimulus_start_time=datetime(2020, 10, 10, 12, 7, 00),
+    stimulus_end_time=datetime(2020, 10, 10, 13, 1, 00),
+    mouse_platform_name="Platform A",
+    active_mouse_platform=False,
+)
+
+etl_job = BergamoEtl(
+    input_source=Path("/directory/of/tiff/files/"),
+    output_directory=Path("/location/to/save/to/"),
+    user_settings=user_settings,
+)
+
+# To crawl through the tiff directory and parse the headers, simply run:
+
+etl_job.run_job()
+
+# If the headers have already been parsed, it's possible to run the steps
```

Comment:
I found this comment a little confusing.  Maybe instead:

```python
# Below is not normally necessary, but shows how to parse ScanImage header strings manually.

Reply:
I updated the comment with the suggestion